grafana/scylla-dash-per-server.json

```diff
@@ -377,7 +377,7 @@
                         "targets": [
                             {
                                 "expr": "avg(seastar_reactor{type=\"gauge\", metric=\"load\"} ) by (instance)",
-                                "intervalFactor": 2,
+                                "intervalFactor": 1,
                                 "refId": "A",
                                 "step": 4
```

Comment:
Yes it's intentional. The step doesn't seem to be  relevant in this change. I don't actually  know what it signifies.